<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDialog</class>
 <widget class="QDialog" name="CustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer</string>
  </property>
  <widget class="QLabel" name="lblLastName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLastName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblFirstName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPhoneNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblUserType">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPhoneNumber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Phone Num:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFirstName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtFirstName</tabstop>
  <tabstop>txtLastName</tabstop>
  <tabstop>txtPhoneNumber</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
